packages/kit/src/client/DraggableFlatList.tsx

```diff
@@ -0,0 +1,57 @@
+import DraggableFlatList, { ScaleDecorator } from 'react-native-draggable-flatlist'
```

Comment:
Everything that uses React and is client-side goes into this folder, otherwise into "server". Easier to differentiate that way.